codes/classical/bits/repetition code.yml

```diff
@@ -0,0 +1,58 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+code_id: repetition
+physical: bits
+logical: bits
+
+name: 'Binary repetition code'
+
+introduced: '\cite{doi:10.1201/9781315147901}'
+
+description: 'This code encodes 1-bit information into a \(n\)-bit string. \(n\) needs to be an odd number, since receiver will pick the majority at decoding stage. Its basic idea is to increase the code distance by repeating the logical information several times. It is a \((n,1)\)-Hamming code.'
+
+protection: 'Protects against bit-flip and erasure up to \(\frac{n-1}{2}\)-bit error. Bit-flip error and erasure error are the common in classical communication processes.'
```

Comment:
Replace the above description with "Protects against (n-1)/2 bit flips or against n-1 erasures."